Apps/W1/Shopify/app/src/Companies/Interfaces/ShpfyICompanyMapping.Interface.al

```diff
@@ -6,4 +6,6 @@ namespace Microsoft.Integration.Shopify;
 interface "Shpfy ICompany Mapping"
```

Comment:
This is a public interface.
it means that your procedure addition is a breaking change to anyone who has currently implemented this codeunit. You have to make an enum extension to gracefully handle this addition